ui/app/pages/index.us

```diff
@@ -59,5 +59,6 @@
 
 
     <script type="text/javascript" src="/js/app.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js" integrity="sha384-b/U6ypiBEHpOf/4+1nzFpr53nxSS+GLCkfwBdFNTxtclqqenISfwAzpKaMNFNmj4" crossorigin="anonymous"></script>
```

Comment:
Are you using anything from this library? If so, it needs to go in the vendor directory. We don't want to load 3rd party content.

Reply:
I made use of it when I used the previous design. But not anymore, so I will do away with it.